Marlin/SanityCheck.h

```diff
@@ -957,19 +957,40 @@ static_assert(1 >= 0
     #error "RGB_LED requires RGB_LED_R_PIN, RGB_LED_G_PIN, and RGB_LED_B_PIN."
   #elif ENABLED(RGBW_LED)
     #error "Please enable only one of RGB_LED and RGBW_LED."
+  #elif ENABLED(LEDSTRIP)
+    #error "Please enable only one of RGB_LED and LEDSTRIP."
```

Comment:
This would be better…
```cpp
    #elif ENABLED(RGBW_LED) || ENABLED(LEDSTRIP)
      #error "Please enable only one of RGB_LED, RGBW_LED, and LEDSTRIP."
    . . .
```